spec/unit/active_node/scope_eval_context_spec.rb

```diff
@@ -0,0 +1,9 @@
+describe Neo4j::ActiveNode::Scope::ScopeEvalContext do
+  describe 'method missing' do
+    let(:target) { double('target') }
+    let(:query_proxy) { double(base: 'Chunky') }
+    subject { described_class.new(target, query_proxy).base }
+
+    it { is_expected.to eq('Chunky') }
```

Comment:
@amitsuryavanshi this is not testing the method `method_missing` you changed. A proper test should test that when you call a nonexisting method on `ScopeEvalContext` that method is called on `target` or `query_proxy`. Furthermore if you call a method which does not even exist on those 2 then a `method_missing` is called on one of those.